### PR TITLE
CI improvements

### PR DESCRIPTION
**Pull request by rpiszczatowski**

### CODE REVIEW

File: packages/oracle-node/src/fetchers/twelve-data/twelve-data-fetchers-config.json [hunk 1]
```diff
@@ -91,7 +91,7 @@
       "IGMS": "IGMS",
       "GME": "GME",
       "AMC": "AMC",
-      "APE": "APE",
+      "APE": "APE/USD",
```
Comment (koonopek) [inline packages/oracle-node/src/fetchers/twelve-data/twelve-data-fetchers-config.json:94]: sure?

File: .github/workflows/integration-test-template.yml [hunk 2]
```diff
@@ -20,14 +20,15 @@ jobs:
         run: yarn --frozen-lockfile --immutable
       - name: Build all
         run: |
-          echo 0 > STATUS_TMP 
-          cd packages/cache-service && yarn build || echo 1 > STATUS_TMP  &
-          cd packages/evm-connector && yarn compile && yarn build || echo 2 > STATUS_TMP  &
-          cd packages/oracle-node && yarn build || echo 3 > STATUS_TMP  &
-          cd packages/on-chain-relayer && yarn compile && yarn build || echo 4 > STATUS_TMP  &
+          STATUS_TMP=$(pwd)/status.tmp
```
[the remark this answers is not in the document]
Comment (rpiszczatowski) [reply .github/workflows/integration-test-template.yml:23]: STATUS_TMP is a relative path. The previous version created this file in a subdirectory on failure and then checked the status in the top-level directory. I observed a couple of times when the build failed (based on logs) but the 'build all' step status was successful. Now it does fail.

File: .github/workflows/oracle-arbitrum-dry-run-test.yml [hunk 3]
```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Build
         run: yarn build
       - name: Run dry run test
-        run: yarn test:arbitrum-dry-run
+        run: yarn test:arbitrum-dry-run 2>&1 | tee logs.log; test ${PIPESTATUS[0]} -eq 0
```
[the remark this answers is not in the document]
Comment (rpiszczatowski) [reply .github/workflows/oracle-arbitrum-dry-run-test.yml:22]: You can verify for the third time that I verified a couple of times that it fails by looking in previous runs of GH actions in this PR :)

File: .github/workflows/oracle-arbitrum-dry-run-test.yml [hunk 4]
```diff
@@ -28,3 +28,6 @@ jobs:
           TWELVE_DATA_API_KEY: ${{ secrets.TWELVE_DATA_API_KEY }}
           KAIKO_API_KEY: ${{ secrets.KAIKO_API_KEY }}
           SKIPPED_SOURCES: '["binance", "binancecoinm", "binanceusdm", "bybit"]'
+      - name: Create summary
+        if: ${{ !cancelled() }}
+        run: scripts/filter-logs.sh logs.log >> $GITHUB_STEP_SUMMARY || true
```
[the remark this answers is not in the document]
Comment (rpiszczatowski) [reply .github/workflows/oracle-arbitrum-dry-run-test.yml:33]: This one always succeeds and yes I've verified it. https://github.com/redstone-finance/redstone-oracles-monorepo/actions/runs/6382941023

File: packages/oracle-node/src/fetchers/twelve-data/twelve-data-fetchers-config.json (hunk 1 above)
Comment (rpiszczatowski) [reply packages/oracle-node/src/fetchers/twelve-data/twelve-data-fetchers-config.json:94]: Agreed with @cehali

**Review by koonopek — APPROVED** [review]
Great job!

File: .github/workflows/integration-test-template.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Lukasz2891) [reply .github/workflows/integration-test-template.yml:23]: that was one of the acceptance criterias ;)

**Pull request merged**